packages/react/src/converter/react-wrapper.tsx

```diff
@@ -44,7 +44,7 @@ export function createReactComponent<BaseComponent extends HTMLElement>(elementN
     }
 
     _propIsReservedReactProp(prop) {
-      const reactProperties = ['children', 'localName', 'ref', 'style', 'className'];
+      const reactProperties = ['children', 'localName', 'ref', 'style'];
```

Comment:
I feel like the only thing we are missing is a test here. Both to verify the fix is working as expected as well as to ensure this isn't broken in the future.

Reply:
@mathisscott, I added a test, but it's _definitely_ a bad one: it passes before and after my fix. Would you be able to help me figure out a better test case for this?

Reply:
@astorije 
I pulled down your branch and verified locally that applying a classname to the wrapper component does, in fact, update the CSS custom properties as expected inside the web component. My assumption was that this is a big part of the use case for this functionality. So given that I've verified that and we now have a test in place, I think we are 👍